packages/worker/src/utilities/email.ts

```diff
@@ -24,6 +25,12 @@ type SendEmailOpts = {
   cc?: boolean
   bcc?: boolean
   automation?: boolean
+  addInvite?: boolean
```

Comment:
would be:
```
invite?: {
  startTime: Date
  endTime: Date
  summary: string
  location: string
  url: string
}
```

improves typing as the required properties for a meeting can now properly be required.